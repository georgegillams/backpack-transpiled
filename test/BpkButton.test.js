import React from 'react';
import { render } from '@testing-library/react';
import BpkButton from 'backpack-transpiled/bpk-component-button';

describe('<BpkButton />', () => {
  it('should render correctly', () => {
    const { container } = render(<BpkButton>This is a button</BpkButton>);

    expect(container).toMatchSnapshot();
  });
});
