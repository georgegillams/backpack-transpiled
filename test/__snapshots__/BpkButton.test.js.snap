// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BpkButton /> should render correctly 1`] = `
<div>
  <button
    class="bpk-button"
    type="button"
  >
    This is a button
  </button>
</div>
`;
